packages/core/src/indexing/postgres/__tests__/postgres-index-api.test.ts

```diff
@@ -134,6 +135,7 @@ describe('indexStream', () => {
   const STREAM_CONTENT = {
     model: MODELS_TO_INDEX[0],
     streamID: StreamID.fromString(STREAM_ID_B),
+    streamContent: STREAM_TEST_DATA_PROFILE,
     controller: CONTROLLER,
     lastAnchor: null,
   }
```

Comment:
I think we should also have a test that builds an index on one of the fields in the content JSON and does a query against the postgres database to make sure it works.  That way we know that we've definitely provided the data in a way that Postgres JSONB indexing can actually work.

Reply:
Yeah I agree. I was actually thinking about that after I opened the PR and will add that as well.